core/server/proxy/src/main/java/alluxio/proxy/s3/S3ErrorCode.java

```diff
@@ -23,15 +23,19 @@
    */
   public static final class Name {
     public static final String BAD_DIGEST = "BadDigest";
+    public static final String BAD_REQUEST = "BadRequest";
```

Comment:
This is not an existing [S3 error code](), however it _was_ an error code I observed from the `aws-cli`.